### DISCUSSION TITLE
How to add videos to github notebook

### DISCUSSION THREAD

**Discussion by drlawson** [RobertsLab/resources#1355] [Q&A] [answered]
Hi all, in lab meeting today we discussed a bit about adding videos to a github notebook, and finding a better way to add them than putting a link to the mp4 file in github. If anyone has any suggestions on how to do this, that would be great. I included a link to my notebook post so you can see what I did as a rough draft. The videos in this notebook entry aren't quite finalized yet, as I misspoke in a couple sections and need to fix that, so keep in mind that I will be needing to switch these videos out in the next couple days. Thanks for your help!! :) 

https://drlawson.github.io/tut-3-videos/

**Comment by kubu4** [comment]
Easiest is probably to upload to YouTube and use `<iframe>` and YouTube embed URL. See this for more info:

https://stackoverflow.com/questions/34708476/jekyll-blog-hosted-in-github-how-to-insert-youtube-embeddings

**Comment by sr320** [comment, marked as answer]
Well actually..

if you have it on a server (like gannet, or anywhere you can grab a url) you can use code such as

```
<html>
<body>

<video width="320" height="240" controls>
  <source src="https://gannet.fish.washington.edu/seashell/snaps/LightningTalks.mp4" type="video/mp4">
  Your browser does not support the video tag.
</video>

</body>
</html>
```
